js/customer.js

```diff
@@ -0,0 +1,95 @@
+        }else if(validName(name) && validPhoneNumber(phoneNumber) && !validEmail(email)) {           
+                    email.style.background = '#f00';
+        // If the email and phone number are valid but name not execute this code
+        /*==================================================================*/
+        }else  if(!validName(name) && validPhoneNumber(phoneNumber) && validEmail(email)) {
+                    name.style.background = '#f00';
+        // If the name is valid but email and phone number are not execute this code
+        /*==================================================================*/
+        }else if(validName(name) && !validPhoneNumber(phoneNumber) && !validEmail(email)) {
+                    email.style.background = '#f00';
+                    phoneNumber.style.background = '#f00';
+        // If the email is valid but name and phone number are not execute this code
+        /*==================================================================*/
+        } else if(!validName(name) && !validPhoneNumber(phoneNumber) && validEmail(email)) {
+                    name.style.background = '#f00';
+                    phoneNumber.style.background = '#f00';
+        // If the phone number is valid but email and name are not execute this code
+        /*==================================================================*/
+        }else if(!validName(name) && validPhoneNumber(phoneNumber) && !validEmail(email)) {
+                    name.style.background = '#f00';
+                    email.style.background = '#f00';
+        }else {
+                    alert('error');
+                    return false;
+        }
+
+    });
+}, false);
+/*==================== Start AJAX Get Data===================*/
+window.addEventListener('load', function () {
+    const resivedNews = document.getElementById('resivedNews');
+resivedNews.style.display = 'none';
+const showNews = document.querySelector('#showNews');
+const url = "https://private-e99507-kabaros.apiary-mock.com/news";
+var request = new XMLHttpRequest();
+    showNews.addEventListener('click', function (event) {
+        event.preventDefault();
+        request.onreadystatechange = function () {
+            if (request.readyState === 4 ){
+                if (request.status === 200) {
+                    const data = JSON.parse(request.responseText);
+                    for (i = 0; i < data.length; i++) {                       
+                        resivedNews.innerHTML += "<h2>"+data[i].title+"</h2>"+"<br>"+ "<p>"+data[i].summary+".</p><br>";
+                    }
+                    resivedNews.style.display = 'block';
+                }else {
+                    console.log();
+                    resivedNews.innerHTML = 'An error occurred during your request: ';
+                } 
+            }         
+        }
+        request.open("GET", url);
+        request.send();
+    });
+
+    const hideNews = document.querySelector('#hideNews');
```

Comment:
the requirement was to use the same link to show/hide .. so when you click first time, it shows the div, then the second time, it hides it. Can you update your code to do that please